.travis.yml

```diff
@@ -0,0 +1,85 @@
+language: c
+dist: trusty
+sudo: false
+group: beta
+
+# To cache doc-building dependencies.
+cache: pip
+
+os:
+  - linux
+  # macOS builds are disabled as the machines are under-provisioned on Travis,
+  # adding up to an extra hour completing a full CI run.
+  #- osx
+
+compiler:
+  - clang
+  - gcc
+
+env:
+  - TESTING=cpython
+
+matrix:
+  allow_failures:
+    - env:
+       - TESTING=coverage
+  include:
+    - os: linux
+      language: python
+      python: 3.5
+      env:
+        - TESTING=docs
+      before_script:
+        - cd Doc
+        - make venv PYTHON=python3
```

Comment:
Since there is no `make venv` on 2.7, just do `pip install -U Sphinx` (or add a 'venv' target, but that's a bit weird since there is no 'venv' module in 2.7).